### PR TITLE
chore: Improve bio and nickname exceeded max length errors

### PR DESCRIPTION
**Pull request by matheusbsilva137**
## Proposed changes (including videos or screenshots)

 - Throw more meaningful errors when setting bio or nickname that exceeds max length (introduce `error-bio-size-exceeded` and `error-nickname-size-exceeded)`.

## Further comments

TC-723

### CODE REVIEW

**Review by KevLehman — COMMENTED** [review]
Can i haz some tests? 😬

File: apps/meteor/app/lib/server/functions/saveUser.js [hunk 1]
```diff
@@ -237,11 +237,16 @@ export async function validateUserEditing(userId, userData) {
 
 const handleBio = (updateUser, bio) => {
 	if (bio && bio.trim()) {
-		if (typeof bio !== 'string' || bio.length > MAX_BIO_LENGTH) {
+		if (typeof bio !== 'string') {
```
Comment (KevLehman) [inline apps/meteor/app/lib/server/functions/saveUser.js:240]: It's kinda weird that we do a `trim` and then we check for the value to be an string tbh

Comment (matheusbsilva137) [reply apps/meteor/app/lib/server/functions/saveUser.js]: Agree. Removed this check from `bio` and `username` in this file

Comment (KevLehman) [reply apps/meteor/app/lib/server/functions/saveUser.js]: For safety i think you can keep them, but will be better placed before trying to call `.trim()` :)

**Review by tapiarafael — APPROVED** [review]

**Review by tapiarafael — DISMISSED** [review]

**Review by tapiarafael — APPROVED** [review]

**Pull request merged**